documentation/staging/content/managing-domains/accessing-the-domain/istio/istio.md

```diff
@@ -52,6 +56,8 @@ The current support for Istio has these limitations:
   and has been tested with single and multicluster
   Istio installations from 1.10 up to 1.13.2.
 
+* For Red Hat OpenShift Service Mesh, continue to use operator version 3.4.  This version of the operator is incompatible with OpenShift Service Mesh, base Istio version implementation which is 1.9.
```

Comment:
This recommendation is unclear to me.  Continue to use operator 3.4, but it won't work?  Since we were able to get the OpenShift environment for our partner to work on the service mesh, I assume that Istio does work.  Please clarify your statement.